GitCommands/RevisionReader.cs

```diff
@@ -428,6 +423,13 @@ long ParseUnixDateTime(ref ReadOnlySpan<byte> array)
             };
 
             return true;
+
+            static void ParseAssert(string? message)
+            {
+                _noOfParseError++;
+                Debug.Assert(_noOfParseError > 1, message);
```

Comment:
Shall the first error be ignored? Or is the identifier misleading as I guessed above?

Reply:
The first error fails to be asserted, so it is displayed. The other are ignored.

Reply:
Thank you. You are right.